tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
+                                              timestamp=1597324762847, step=1, is_nan=False),
+                                ElasticMetric(key="metric1", value=7.0,
+                                              timestamp=1597324762890, step=2, is_nan=False),
+                                ElasticMetric(key="new_metric", value=7.0,
+                                              timestamp=10, step=0, is_nan=False),
+                                ElasticMetric(key="nan_metric", value=0.0,
+                                              timestamp=10, step=0, is_nan=True),
+                                ElasticMetric(key="inf_metric", value=1.7976931348623157e308,
+                                              timestamp=10, step=0, is_nan=False),
+                                ElasticMetric(key="negative_inf_metric",
+                                              value=-1.7976931348623157e308,
+                                              timestamp=10, step=0, is_nan=False),
+                                ElasticMetric(key="new_metric", value=-10,
+                                              timestamp=20, step=1, is_nan=False)]
+    expected_elastic_latest_metrics = [ElasticLatestMetric(key="metric0", value=20.0,
+                                                           timestamp=1597324762778, step=2,
+                                                           is_nan=False),
+                                       ElasticLatestMetric(key="metric1", value=7.0,
+                                                           timestamp=1597324762890, step=2,
+                                                           is_nan=False),
+                                       ElasticLatestMetric(key="new_metric", value=-10,
+                                                           timestamp=20, step=1, is_nan=False),
+                                       ElasticLatestMetric(key="nan_metric", value=0.0,
+                                                           timestamp=10, step=0, is_nan=True),
+                                       ElasticLatestMetric(key="inf_metric",
+                                                           value=1.7976931348623157e308,
+                                                           timestamp=10, step=0, is_nan=False),
+                                       ElasticLatestMetric(key="negative_inf_metric",
+                                                           value=-1.7976931348623157e308,
+                                                           timestamp=10, step=0, is_nan=False)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```

Comment:
Here only check that you added this metric to the metrics array, and that you updated latest_metric value for this metric